backend/src/main/java/com/htecgroup/skynest/security/WebSecurity.java

```diff
@@ -21,36 +25,44 @@ public class WebSecurity extends WebSecurityConfigurerAdapter {
   private final BCryptPasswordEncoder bCryptPasswordEncoder;
   private final ObjectMapper objectMapper;
 
+  private final UserService userService;
+
   @Override
   protected void configure(HttpSecurity http) throws Exception {
     http.cors()
         .and()
         .csrf()
         .disable()
         .authorizeRequests()
-        .antMatchers(HttpMethod.POST, SecurityConstants.SIGN_UP_URL)
+        .antMatchers(HttpMethod.POST, UrlUtil.POST_URLS_WITHOUT_AUTH)
         .permitAll()
-        .antMatchers(HttpMethod.GET, SecurityConstants.CONFIRM_EMAIL_URL)
+        .antMatchers(HttpMethod.GET, UrlUtil.GET_URLS_WITHOUT_AUTH)
         .permitAll()
-        .antMatchers(HttpMethod.POST, SecurityConstants.PASSWORD_RESET_REQUEST_URL)
+        .antMatchers(HttpMethod.PUT, UrlUtil.PUT_URLS_WITHOUT_AUTH)
         .permitAll()
-        .antMatchers(HttpMethod.PUT, SecurityConstants.PASSWORD_RESET_CONFIRM_URL)
+        .antMatchers(HttpMethod.DELETE, UrlUtil.DELETE_URLS_WITHOUT_AUTH)
```

Comment:
Is this empty for now?

Reply:
It is.

Reply:
Then add this part when you needed, do not leave code that currently doesn't do anything. Pls remove it